core/src/main/java/com/kin/ecosystem/core/util/StringUtil.java

```diff
@@ -54,4 +54,8 @@ public static String getAmountFormatted(String amount) {
 			return amount;
 		}
 	}
+
+	public static boolean isEmpty(String text) {
```

Comment:
TextUtils.isEmpty()?

Reply:
don't what to use android utills in here.